test/config.custom.yaml

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: CC0-1.0
 
 ### CHANGES TO CONFIG.TUTORIAL.YAML ###
-version: 0.4.1
+version: 0.4.2
```

Comment:
I thought about fixing a typo: the current version of the config is `0.4.2`, and it should be probably the same across the scripts?

Reply:
That is probably a refuse maybe; hadn't the number to match the version of the repo?

Reply:
> That is probably a refuse maybe; hadn't the number to match the version of the repo?

Agree that it would be best when there is a match between the configuration and the release versions. But what is we introduce changes into the config when the repo version remains the same? We have discussed the idea to use sub-versions of the config to track (e.g. `0.4.1.1`) that but agreed that it's against semantic versioning. Ideally, it would be great to have a release every time when the config changes, but that is not exactly realistic as far as we don't have an automated release process

Happy to discuss at the next convenience 🙂